data/event.js

```diff
@@ -247,6 +247,17 @@ const events = [{
+  {id: 102,title: 'Advanced Track',allDay: false,start: 1643248800000,end: 1643256000000,location: 'https://ucla.zoom.us/j/3616763997',committee: 'icpc',description: "In ICPC's Advanced Track, we submerge and dive deep into the world of algorithms and data structures, tackling ones often found in harder competitive programming problems. ICPC competitors, competitive programming fanatics, and anyone interested in expanding their programming brain are wholeheartedly invited to come and explore these awesome algorithmic wonders with us.\n\nCome to our second workshop of the quarter this Wednesday at 6:00 to learn about complex string algorithms every competitive programmer needs to know!",links: [{text: 'Facebook Event',href: 'https://fb.me/e/4QVOk38pd',ext: true}]},
+  {id: 103,title: 'ACM-W Internet Scavenger Hunt',allDay: false,start: 1643248800000,end: 1643254200000,location: 'https://ucla.zoom.us/j/93342296948',committee: 'w',description: 'Ready to channel your inner detective 🕵️ Join ACM-W for our first ever Internet Scavenger Hunt, where we scour the world wide web and use those handy CS skills to help you find some of the most elusive webpages on the Internet for awesome prizes 🏆 Some example treasures that we’ll be looking for include:\nA Github page that has been active for the last three days 🖥️\nA URL with 3 consecutive numbers 🔢\nA Wikipedia page with less than 50 occurrences of the word “the” 🌐\n',links: [{text: 'Facebook Event',href: 'https://fb.me/e/49jGXQW8z',ext: true}]},
+  {id: 104,title: 'Hack Sprint Mobile Development Series',allDay: false,start: 1643248800000,end: 1643256000000,location: 'Zoom',committee: 'hack',description: 'With Week 4 coming up fast, we’re hurtling 🏃‍♀️ around the corner to session 3 of Hack Sprint, our beginner-friendly iOS mobile development workshop! What better way to have fun 🤸‍♀️ as we jump into midterm season than learning about mobile app development in Swift! 🚨 Join us Wednesday, January 26th from 6-8 PM PST as talk about views 🔍, view modifiers, states, and bindings! Off you pop, and see you there! 💜\n\nMissed our last session?\nCheck out the recording: https://youtu.be/ayzYEj6OZFA \nHere’s our comprehensive README: http://links.uclaacm.com/hacksprint22-s2',links: [{text: 'Zoom Link', href: 'https://ucla.zoom.us/j/93761587241?pwd=aWlsV3FxRzl6clVhRDQ0RHF4dmN6dz09', ext: true},{text: 'Facebook Event',href: 'https://fb.me/e/1FvZyUmkT',ext: true}]},
```

Comment:
There are two links in the body of the text for this one (and the previous hack sprint one as well)